src/elements/content-open-with/OpenWithButton.js

```diff
@@ -20,7 +20,7 @@ type Props = {
     error: ?any,
     icon?: string,
     isLoading: boolean,
-    onClick: Function,
+    onClick?: Function,
```

Comment:
Is a default value required now? It looks like it's being invoked unguarded below.

Reply:
Can you add an actual function type to this instead  e.g. ```(data:Type) => void```